app/views/idv/shared/_verify.html.erb

```diff
@@ -0,0 +1,119 @@
+<%#
+locals:
+  pii - user's information
+  remote_identity_proofing - true if we're doing RIDP, false if we're doing IPP
+  step_url - generator for step URLs
+%>
+<% title t('titles.doc_auth.verify_info') %>
+
+<%= render PageHeadingComponent.new.with_content(t('doc_auth.headings.verify')) %>
+<div class='margin-top-4 margin-bottom-2'>
+  <% if remote_identity_proofing %>
+    <div>
+      <%= "#{t('doc_auth.forms.first_name')}: #{pii[:first_name]}" %>
+    </div>
+    <div>
+      <%= "#{t('doc_auth.forms.last_name')}: #{pii[:last_name]}" %>
+    </div>
+    <div class='padding-bottom-1'>
+      <%= "#{t('doc_auth.forms.dob')}: #{pii[:dob]}" %>
+    </div>
+  <% else %>
+    <div class="grid-row grid-gap grid-gap-2 padding-bottom-1 padding-top-1">
+      <div class='grid-col-fill'>
+        <div>
+          <%= "#{t('doc_auth.forms.first_name')}: #{pii[:first_name]}" %>
+        </div>
+        <div>
+          <%= "#{t('doc_auth.forms.last_name')}: #{pii[:last_name]}" %>
+        </div>
+        <div>
+          <%= "#{t('doc_auth.forms.dob')}: #{pii[:dob]}" %>
+        </div>
+        <div class='padding-bottom-1'>
+          <%= "#{t('doc_auth.forms.id_number')}: #{pii[:state_id_number]}" %>
+        </div>
+      </div>
+      <div class='grid-auto'>
+        <%= button_to(
+              step_url.call(step: :redo_state_id),
+              method: :put,
+              class: 'usa-button usa-button--unstyled',
+              'aria-label': t('doc_auth.buttons.change_state_id_label'),
+            ) { t('doc_auth.buttons.change_label') } %>
+      </div>
+    </div>
+  <% end %>
```

Comment:
Since these details are mostly the same, and since `grid-col-fill` will automatically take up as much space as it can, maybe we can just have one set of markup, and optionally add in the ID number and "Change" button for IPP:

```suggestion
  <div class="grid-row grid-gap grid-gap-2 padding-bottom-1 padding-top-1">
    <div class="grid-col-fill">
      <div>
        <%= "#{t('doc_auth.forms.first_name')}: #{pii[:first_name]}" %>
      </div>
      <div>
        <%= "#{t('doc_auth.forms.last_name')}: #{pii[:last_name]}" %>
      </div>
      <div>
        <%= "#{t('doc_auth.forms.dob')}: #{pii[:dob]}" %>
      </div>
      <% if !remote_identity_proofing %>
        <div>
          <%= "#{t('doc_auth.forms.id_number')}: #{pii[:state_id_number]}" %>
        </div>
      <% end %>
    </div>
    <% if !remote_identity_proofing %>
      <div class="grid-auto">
        <%= button_to(
              step_url.call(step: :redo_state_id),
              method: :put,
              class: 'usa-button usa-button--unstyled',
              'aria-label': t('doc_auth.buttons.change_state_id_label'),
            ) { t('doc_auth.buttons.change_label') } %>
      </div>
    <% end %>
  </div>
```

(I intentionally omitted the `padding-bottom-1` because I don't think it needs to be there based on Figma designs)

Reply:
Thanks, added in [6719404](https://github.com/18F/identity-idp/pull/6514/commits/67194041fee3de894ed3cd98178ebb60cc2f7598)